storage/rocksdb/ha_rocksdb.cc

```diff
@@ -1157,8 +1148,43 @@ static MYSQL_SYSVAR_BOOL(pause_background_work, rocksdb_pause_background_work,
 
 static MYSQL_SYSVAR_BOOL(
     enable_ttl, rocksdb_enable_ttl, PLUGIN_VAR_RQCMDARG,
-    "Disable expired ttl records from being dropped during compaction.",
-    nullptr, rocksdb_set_enable_ttl, TRUE);
+    "Enable expired TTL records to be dropped during compaction.", nullptr,
+    nullptr, TRUE);
+
+static MYSQL_SYSVAR_BOOL(
```

Comment:
I see, something could be done about mysqld--help-notwin as it already filters plugin variables and does not fail for the other debug-only ones, but rocksdb.rocksdb could be enough of a reason to leave them in